lms/djangoapps/courseware/views/views.py

```diff
@@ -956,6 +956,31 @@ def course_about(request, course_id):
         # get prerequisite courses display names
         pre_requisite_courses = get_prerequisite_courses_display(course)
 
+        courses_having_prerequisites = frozenset({course.id})
+        courses_requirements_not_met = get_pre_requisite_courses_not_completed(request.user, courses_having_prerequisites)
+        ecommerce_service = EcommerceService()
+        sku_not_enrollment_in_requirement = None
+        course_requirements = None
+        run_extension_point(
+            'PEARSON_CORE_SORT_ENROLLED_PREREQUISITES',
+            user=request.user,
+            courses_requirements_not_met=courses_requirements_not_met,
+        )
+
+        course_requirements = run_extension_point(
+            'PEARSON_CORE_STUDENT_COURSE_REQUIREMENTS_FOR_COURSEWARE',
+            courses_requirements_not_met=courses_requirements_not_met,
+        )
+
+        sku_not_enrollment_in_requirement = run_extension_point(
+            'PEARSON_CORE_STUDENT_NOT_ENROLLED_IN_REQUIREMENTS',
+            user=request.user,
+            courses_requirements_not_met=courses_requirements_not_met,
+        )
+
+        sku_not_enrollment_in_requirement = \
+            sku_not_enrollment_in_requirement[course.id] if sku_not_enrollment_in_requirement else None
```

Comment:
Please write all the code related to the `'PEARSON_CORE_STUDENT_NOT_ENROLLED_IN_REQUIREMENTS'` function in Pearson Core. @anfbermudezme